src/pixbuf.rs

```diff
@@ -34,8 +34,16 @@ impl Pixbuf {
         unsafe extern "C" fn destroy<T: AsMut<[u8]>>(_: *mut c_uchar, data: *mut c_void) {
```

Comment:
I'd actually prefer instead of panicking in here to make `new_from_mut_slice` return an `Option` if any of the checks is invalid and to use `g_debug` as well.

Or even better: using a `Result<Pixbuf, &str>` where `&str` would tell which assert failed. That'd allow us to not use `g_debug`.

Reply:
The reason for panicking is that passing incorrect values for those arguments is a programming error, and there is nothing the program can do about it afterwards.

The only way in which this function could fail "recoverably" is if `Box::new` OOMs; the underlying `gdk_pixbuf_new_from_data()` cannot fail (it will crash if it can't malloc the GObject, same as the rest of glib, but it doesn't do checked allocations).

Reply:
In addition to that, it's an unsafe function, so I guess it's fine...